config/cli.go

```diff
@@ -118,6 +120,11 @@ func addOrEdit(c *cli.Context, operation configOperation) error {
 		return err
 	}
 
+	oidcParams, err := createOidcParamsFromFlags(c)
```

Comment:
I'm not sure if I got it right, but it looks like the function `createOidcParamsFromFlags` is already called in the functoin `CreateConfigCommandConfiguration` above, so can this call to the function be removed?